namespace-provisioner/how-tos.hbs.md

```diff
@@ -0,0 +1,254 @@
+
+Any changes to the resources that have the `...no-overwrite` annotation are not overwritten by the **`provisioner`** kapp App controlling the resource provision. If Platform/App Operators want to restore the default state of those resources, they can simply delete them and the [provisioner](about.hbs.md#provisioner-carvel-app) kapp App will re-create them in their initial default state.
+
+The kapp App controlling the resource provisioning has an sync interval of 10 minutes. To manually force the reconciliation of the resources (for example: Deleting a resource so that it can be created into its default initial state), you can use the [Carvel kctrl](https://carvel.dev/blog/kctrl-release-blog/) CLI to “kick” the “provisioner” app reconciliation. 
+
+You can initiate the "kick" by running the following command:
+
+```bash
+kctrl app kick --app provisioner -n tap-namespace-provisioning -y
+```
+
+</br>
+
+---
+
+</br>
+
+### <a id="control-desired-namespaces"></a>Control the `desired-namespaces` ConfigMap via GitOps
+
+Prerequisites:
+
+- Namespace Provisioner package is installed and successfully reconciled.
+- [**`controller`** tap value key](install.hbs.md#customized-installation) is set to “false” (Default is “true”).
+- The registry-credentials secret referred by the Tanzu Build Service is added to tap-install and exported to all namespaces. If you don’t want to export this secret to all namespaces for any reason, you will have to go through an additional step to create this secret in the namespace.
+
+Users can choose to maintain the [`desired-namespaces`](about.hbs.md#nsp-component-desired-namespaces-configmap) ConfigMap in their git repository and opt not to use the namespace-provisioner controller. Users can use their choice of GitOps tools to override the [`desired-namespaces`](about.hbs.md#nsp-component-desired-namespaces-configmap) ConfigMap in the tap-namespace-provisioning namespace. 
+
+>**NOTE:** you need to set the “controller” tap value key to “false” (Default is “true”) for namespace provisioner in tap-values.yaml file. The [`controller`](install.hbs.md#customized-installation) value in tap-values decides whether the Out-of-the-box controller is installed and used for maintaining the list of desired-namespaces or not.</br>
+If the controller is installed as part of the package, when it reconciles, it will remove all namespaces that are not properly labeled with the [`namespace_selector`](install.hbs.md#customized-install) specified in the tap values (Default label selector is **`apps.tanzu.vmware.com/tap-ns=""`**). 
+
+Use the following snippet as reference for the desired-namespaces ConfigMap that you can put on your git repository.
+Desired-namespaces.yaml ([Link to sample repo file](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/namespace-provisioner-gitops-examples/desired-namespaces/gitops-managed-desired-namespaces.yaml))
+
+```yaml
```

Comment:
That is that way looking to avoid going out of the page to copy/paste content but the link was updated without update the docs, i think it is better to update the snipper